ouroboros-consensus-test/test-consensus/Test/Consensus/HardFork/Combinator.hs

```diff
@@ -22,7 +22,12 @@
 module Test.Consensus.HardFork.Combinator (tests) where
 
 import qualified Data.Map.Strict as Map
+import           Data.SOP.Counting
+import           Data.SOP.InPairs (RequiringBoth (..))
+import qualified Data.SOP.InPairs as InPairs
```

Comment:
Probably not for this PR, but I wonder if it'd make sense to have one `Data.SOP` module that would re-export all these symbols.